apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/internal/SortedInputFieldMapWriter.kt

```diff
@@ -1,139 +0,0 @@
-package com.apollographql.apollo3.cache.normalized.internal
-
-import com.apollographql.apollo3.api.CustomScalar
-import com.apollographql.apollo3.api.internal.InputFieldMarshaller
-import com.apollographql.apollo3.api.internal.InputFieldWriter
-import com.apollographql.apollo3.api.internal.Throws
-import okio.IOException
-
-class SortedInputFieldMapWriter() : InputFieldWriter {
-  private val buffer = mutableMapOf<String, Any?>()
-
-  fun map(): Map<String, Any?> {
-    return buffer.toList().sortedBy { it.first }.toMap()
```

Comment:
Was sorting required? This PR changes this by serializing the Input Objects in the order of their input fields definition. I don't think it's an issue but maybe I'm missing something?